src/components/SideMenu.vue

```diff
@@ -8,57 +10,73 @@
 
     <!--TABS-->
     <div class="flex flex-col text-lg font-medium text-black space-y-3">
-
-      <div class="flex flex-row items-center hover:cursor-pointer hover:text-primary-grey">
-        <SearchCircleIcon class="w-4 h-4"/>
+      <div
+        class="flex flex-row items-center hover:cursor-pointer hover:text-primary-grey"
+      >
+        <SearchCircleIcon class="w-4 h-4" />
         <RouterLink to="/search">
           <p class="ml-3">search</p>
         </RouterLink>
       </div>
 
-      <div class="flex flex-row items-center hover:cursor-pointer hover:text-primary-grey">
-        <PlusCircleIcon class="w-4 h-4"/>
+      <div
+        class="flex flex-row items-center hover:cursor-pointer hover:text-primary-grey"
+      >
+        <PlusCircleIcon class="w-4 h-4" />
         <RouterLink to="/add-post">
           <p class="ml-3">add post</p>
         </RouterLink>
       </div>
 
-      <div class="flex flex-row items-center hover:cursor-pointer hover:text-primary-grey">
-        <CollectionIcon class="w-4 h-4"/>
+      <div
+        class="flex flex-row items-center hover:cursor-pointer hover:text-primary-grey"
+      >
+        <CollectionIcon class="w-4 h-4" />
         <RouterLink to="/my-posts">
           <p class="ml-3">my posts</p>
         </RouterLink>
       </div>
 
-      <div class="flex flex-row items-center hover:cursor-pointer hover:text-primary-grey">
-        <HeartIcon class="w-4 h-4"/>
+      <div
+        class="flex flex-row items-center hover:cursor-pointer hover:text-primary-grey"
+      >
+        <HeartIcon class="w-4 h-4" />
         <RouterLink to="/favourite-posts">
           <p class="ml-3">favourites</p>
         </RouterLink>
       </div>
 
-      <div class="flex flex-row items-center hover:cursor-pointer hover:text-primary-grey">
-        <BellIcon class="w-4 h-4"/>
-        <p class="ml-3">notifications</p>
+      <div
+        class="flex flex-row items-center hover:cursor-pointer hover:text-primary-grey"
+      >
+        <BellIcon class="w-4 h-4" />
+        <RouterLink to="/notifications">
+          <p class="ml-3">notifications</p>
```

Comment:
it would be great to show here that we have notifications, like a circle with a number, or at least other color of an icon